crates/re_log/src/log_web.rs

```diff
@@ -0,0 +1,78 @@
+/// Implements [`log::Log`] to log messages to `console.log`, `console.warn`, etc.
+pub struct WebLogger {
+    filter: log::LevelFilter,
+}
+
+impl WebLogger {
+    pub fn new(filter: log::LevelFilter) -> Self {
+        Self { filter }
+    }
+
+    /// Install this logger as the global logger.
+    pub fn init(filter: log::LevelFilter) -> Result<(), log::SetLoggerError> {
+        log::set_max_level(filter);
+        log::set_boxed_logger(Box::new(Self::new(filter)))
+    }
+}
+
+impl log::Log for WebLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        if metadata.target().starts_with("wgpu") || metadata.target().starts_with("naga") {
+            // TODO(emilk): remove once https://github.com/gfx-rs/wgpu/issues/3206 is fixed
+            return metadata.level() <= log::LevelFilter::Warn;
+        }
```

Comment:
Can we think of a good way to opt in to those logs? There are some situations where I want/need them. 🤔